src/test/java/homeworks/POM/Pages/BasePageHmw.java

```diff
@@ -0,0 +1,26 @@
+package homeworks.POM.Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePageHmw {
+    WebDriver driver;
+    WebDriverWait wait;
+    Actions actions;
```

Comment:
**I recommend:** make the variables protected, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

For example

```
    protected WebDriver driver;
    protected WebDriverWait wait;
    protected Actions actions;
```